File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraReports.Design;
using DevExpress.XtraReports.UserDesigner;
using System.Reflection;
using System.ComponentModel.Design;
using DevExpress.XtraReports.UI;

namespace RepStyleSheets {
    public partial class Form1 : Form {
        XtraReport1 report;
        
        public Form1() {
            InitializeComponent();

            report = new XtraReport1();
        }

        private void Form1_Load(object sender, EventArgs e) {
            report.ApplayStyles();
        }

        private void button1_Click(object sender, EventArgs e) {
            XRDesignFormEx designerForm = new XRDesignFormEx();
            designerForm.OpenReport(report);
            designerForm.DesignPanel.ExecCommand(ReportCommand.ShowPreviewTab);
            designerForm.ShowDialog();
            designerForm.OpenReport((XtraReport)null);
            designerForm.Dispose();
        }

        private void button2_Click(object sender, EventArgs e) {
            if (saveFileDialog1.ShowDialog() == DialogResult.OK)
                report.StyleSheet.SaveToFile(saveFileDialog1.FileName);
        }

        private void button3_Click(object sender, EventArgs e) {
            if (openFileDialog1.ShowDialog() == DialogResult.OK)
                report.StyleSheet.LoadFromFile(openFileDialog1.FileName);

            report.ApplayStyles();
        }

        private void button4_Click(object sender, EventArgs e) {
            report.ResetStyles();
        }

        private void button5_Click(object sender, EventArgs e) {
            XRDesignFormEx designerForm = new XRDesignFormEx();
            designerForm.OpenReport(report);

            StyleSheetEditorForm ssef = new StyleSheetEditorForm(report, null);

            ssef.ShowDialog();

            designerForm.OpenReport((XtraReport)null);
            designerForm.Dispose();
        }
    }
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraReports.Design
Imports DevExpress.XtraReports.UserDesigner
Imports System.Reflection
Imports System.ComponentModel.Design
Imports DevExpress.XtraReports.UI

Namespace RepStyleSheets
	Partial Public Class Form1
		Inherits Form
		Private report As XtraReport1

		Public Sub New()
			InitializeComponent()

			report = New XtraReport1()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			report.ApplayStyles()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			Dim designerForm As New XRDesignFormEx()
			designerForm.OpenReport(report)
			designerForm.DesignPanel.ExecCommand(ReportCommand.ShowPreviewTab)
			designerForm.ShowDialog()
			designerForm.OpenReport(CType(Nothing, XtraReport))
			designerForm.Dispose()
		End Sub

		Private Sub button2_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button2.Click
			If saveFileDialog1.ShowDialog() = System.Windows.Forms.DialogResult.OK Then
				report.StyleSheet.SaveToFile(saveFileDialog1.FileName)
			End If
		End Sub

		Private Sub button3_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button3.Click
			If openFileDialog1.ShowDialog() = System.Windows.Forms.DialogResult.OK Then
				report.StyleSheet.LoadFromFile(openFileDialog1.FileName)
			End If

			report.ApplayStyles()
		End Sub

		Private Sub button4_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button4.Click
			report.ResetStyles()
		End Sub

		Private Sub button5_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button5.Click
			Dim designerForm As New XRDesignFormEx()
			designerForm.OpenReport(report)

			Dim ssef As New StyleSheetEditorForm(report, Nothing)

			ssef.ShowDialog()

			designerForm.OpenReport(CType(Nothing, XtraReport))
			designerForm.Dispose()
		End Sub
	End Class
End Namespace